packages/server/events-processing-platform/domain/organization_plan/command/create_org_plan_command.go

```diff
@@ -0,0 +1,24 @@
+package command
```

Comment:
please remove all the commands classes. please take a look at how the invoice events are build
GRPC service -> creates events by skipping the commands

Reply:
Alternative to reduce amount of event/request handlers,
 check  **OrganizationRequestHandler** in _packages/server/events-processing-platform/domain/organization/request_handler.go_
 single handling of request and in aggregate.HandleRequest by type provide specific implementation